vcpkg_version.txt

```diff
@@ -1 +1 @@
-https://github.com/microsoft/vcpkg.git 2021.04.30
+https://github.com/microsoft/vcpkg.git 4dc864e2401f2ed3230c9042a4dd56f6d1c30360
```

Comment:
This is the commit on `master` right after https://github.com/microsoft/vcpkg/pull/17975 was merged.